src/_locales/pl-PL.json

```diff
@@ -0,0 +1,96 @@
+  "lastfm-login-not-authorized": "Nie zalogowany",
+  "lastfm-logout-button-text": "Wyloguj się z Last.FM",
+
+  "lyrics-failed-message": "Niestety nie znaleziono tekstu tego utworu",
+  "lyrics-loading-message": "Wczytywanie tekstu utworu...",
+  "lyrics-no-song-message": "Żaden utwór nie jest teraz odtwarzany",
+
+  "message-pausing-after-song": "Włączono wstrzymanie po tym utworze.",
+  "message-pausing-after-song-button": "Wyłączono wstrzymanie po tym utworze",
+  "message-uncaught-error": "Wystąpił niespodziewany błąd w aplikacji GPMDP.",
+  "message-uncaught-error-button": "Kliknij tutaj aby zgłosić ten błąd na stronie GitHub",
+
+  "modal-confirmTray-title": "Chcieliśmy tylko uprzedzić",
+  "modal-confirmTray-content": "Zamknięcie głównego okna aplikacji (czyli to, co właśnie próbowałeś zrobić) w rzeczywistości zminimalizuje odtwarzacz do systemowego paska narzędzi. Możesz zmienić to ustawienie w oknie Ustawień Aplikacji.",
+  "modal-confirmUpdate-title": "Dostępna aktualizacja",
+  "modal-confirmUpdate-content": "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy zaktualizowanie aplikacji najszybciej jak to możliwe, żeby zapewnić najlepsze wrażenia z korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!",
+  "modal-confirmUninstall-title": "Jest taka sprawa...",
+  "modal-confirmUninstall-content": "Wykryliśmy w Twoim systemie starszą wersję GPMDP. Jeśli właśnie zaktualizowałeś aplikację, to jest to zupełnie normalne. Teraz odinstalujemy starszą wersję. To \"może\" wymagać uprawnień administratora, proszę udziel ich abyśmy mogli posprzątać ten bałagan.",
+  "modal-goToURL-title": "Otwórz Google Play Music aby przejść do:",
+  "modal-welcome-title": "Witaj w nowej wersji",
+  "modal-welcome-content": "Mamy nadzieję, że polubisz nową wersję!",
+
+  "playback-label-next-track": "Następny utwór",
+  "playback-label-play-pause": "Odtwórz / Wstrzymaj",
+  "playback-label-previous-track": "Poprzedni utwór",
+  "playback-label-stop": "Zatrzymaj",
+  "playback-label-thumbs-down": "Nie lubię",
+  "playback-label-thumbs-up": "Podoba mi się",
+  "playback-label-volume-down": "Zmniejsz głośność",
+  "playback-label-volume-up": "Zwiększ głośność",
+
+  "settings-option-custom-theme": "Własny styl",
+  "settings-option-custom-theme-choose-color": "Wybierz kolor szczegółów dla własnego stylu",
+  "settings-option-custom-theme-choose-custom": "Albo wybierz",
+  "settings-option-custom-theme-custom-color": "Własny kolor",
+  "settings-option-custom-theme-dark": "Ciemny",
+  "settings-option-custom-theme-light": "Jasny",
+
+  "settings-option-enable-api": "Włącz API odtwarzania",
+  "settings-option-enable-api-json": "Włącz API JSON",
+  "settings-option-enable-api-details": "To pozwala innym aplikacjom współpracować z GPMDP",
+  "settings-option-enable-system-borders": "Użyj systemowych ramek okna aplikacji",

```

Comment:
For me it isn't clear enought. Better maybe is "Użyj stylu systemowego dla okna aplikacji". It's better when there is added information about style not some system changes etc. It isn't in like "Style" section so there isn't any context what mean "systemowe ramki". It's sound like programming nature of switch, not style in Polish version.